tutorials/RichDisplay.md

```diff
@@ -0,0 +1,87 @@
+[`RichDisplay`](https://klasa.js.org/RichDisplay.html) allows you to create a paginated menu that users of your bot will be able to browse by using reaction-based navigation.
+
+The most simple working example can achieved by this code:
+
+```javascript
+const { Command, RichDisplay } = require('klasa');
+
+module.exports = class extends Command {
+	constructor(...args) { super(...args, { description: 'Test RichDisplay' }); }
+
+	async run(msg) {
+		return new RichDisplay()
+			.addPage(new Embed())
+			.run(await msg.send('Loading...'));
+	}
+};
+```
+
+A more complex example with an actual usage is this:
+
+```javascript
+const display = new RichDisplay(new this.client.methods.Embed()
+	.setColor(0x673AB7)
+	.setAuthor(this.client.user.name, this.client.user.avatarURL)
```

Comment:
avatarURL is a method, not a property.
https://discord.js.org/#/docs/main/master/class/ClientUser?scrollTo=avatarURL